hexrd/ui/main_window.py

```diff
@@ -294,6 +296,7 @@ def setup_connections(self):
         HexrdConfig().raw_masks_changed.connect(self.update_all)
         HexrdConfig().enable_canvas_toolbar.connect(
             self.on_enable_canvas_toolbar)
+        HexrdConfig().tab_images_changed.connect(self.active_canvas_changed)
```

Comment:
This will be connected to `update_drawn_mask_line_picker_canvas()`, since the tab images changing doesn't necessarily mean that the active canvas has changed.

Reply:
Fixed